src/main/java/org/candlepin/resteasy/filter/StoreFactory.java

```diff
@@ -15,6 +15,7 @@
 package org.candlepin.resteasy.filter;
```

Comment:
OwnerStore is still using member injection on line 107.

(Sorry for weird comment placement. Github won't let me comment on the line directly)

Reply:
I believe the use of static class OwnerStore (and UserStore) keeps us from using constructor injection.